README.md

```diff
@@ -1,12 +1,15 @@
-Yii PHP Framework Version 2
-===========================
+<p align="center">
+    <a href="http://www.yiiframework.com/" target="_blank">
+        <img src="http://static.yiiframework.com/files/logo/yii.png" width="400" alt="Yii Framework" />
+    </a>
+</p>
 
-Thank you for choosing Yii 2 - a modern PHP framework designed for professional Web development.
+Yii 2 is a modern framework designed to be solid foundation for your PHP application.
 
-Yii 2 is a complete rewrite of its previous version Yii 1.1 which is one of the most popular PHP frameworks.
-Yii 2 inherits the main spirit behind Yii for being simple, fast and highly extensible.
-Yii 2 requires PHP 5.4 and embraces the best practices and protocols found in modern Web application development.
+It is fast, secure and efficient and works right out of the box using reasonable defaults.
```

Comment:
Means instead of having to configure every aspect of the framework from scratch it is pre-configured for you with some default which are mostly fine.